dts/bindings/sensor/st,iis2dlpc-common.yaml

```diff
@@ -44,3 +44,76 @@ properties:
         - 2 # Low Power M3
         - 3 # Low Power M4
         - 4 # High Performance
+
+    # tap and tap-tap configuration section
+    # All default values are selected to match the power-up values.
+    # tap and tap-tap events can be generated on INT1 only.
+
+    tap-mode:
+      type: int
+      required: false
+      default: 0
+      description: Tap mode. Default is power-up configuration.
+      enum:
+        - 0 # Only Single Tap
+        - 1 # Single and Double Tap
+
+    tap-thsx:
+      type: int
+      required: false
+      default: 0x0
+      description: Tap X-axis threshold. Default is power-up configuration.
+        (values range from 0x00 to 0x1F)
+        Threshold to start the tap-event detection procedure on the X-axis.
+        Threshold values for each axis are unsigned 5-bit corresponding
+        to a 2g acceleration full-scale range.
+
+    tap-thsy:
+      type: int
+      required: false
+      default: 0x0
+      description: Tap Y-axis threshold. Default is power-up configuration.
+        (values range from 0x00 to 0x1F)
+        Threshold to start the tap-event detection procedure on the Y-axis.
+        Threshold values for each axis are unsigned 5-bit corresponding
+        to a 2g acceleration full-scale range.
+
+    tap-thsz:
+      type: int
+      required: false
+      default: 0x0
+      description: Tap Z-axis threshold. Default is power-up configuration.
+        (values range from 0x00 to 0x1F)
+        Threshold to start the tap-event detection procedure on the Z-axis.
+        Threshold values for each axis are unsigned 5-bit corresponding
+        to a 2g acceleration full-scale range.
+
+    tap-shock:
+      type: int
+      required: false
+      default: 0x0
+      description: Tap shock value. Default is power-up configuration.
+        (values range from 0x0 to 0x3)
+        This register represents the maximum time of an over-threshold signal
+        detection to be recognized as a tap event. Where 0 equals 4*1/ODR and
+        1LSB = 8*1/ODR.
+
+    tap-ltcy:
```

Comment:
maybe spell out `latency`?

Reply:
Sounds good. Since my native language (italian) is so rich of vowels I have developed a special art in cutting them out and using only consonant to have shorter names. But here seems too much in fact. :)